domain/domain.go

```diff
@@ -338,11 +338,16 @@ func (do *Domain) loadSchemaInLoop(lease time.Duration) {
 			if err != nil {
 				log.Errorf("[ddl] reload schema in loop err %v", errors.ErrorStack(err))
 			}
-		case <-syncer.GlobalVersionCh():
+		case _, ok := <-syncer.GlobalVersionCh():
```

Comment:
`<-syncer.GlobalVersionCh()` is likely to be `nil` here.

Reply:
Yes, it may be `nil` here. Then This case doesn't be entered.